src/ray/core_worker/core_worker.cc

```diff
@@ -80,7 +80,8 @@ CoreWorker::CoreWorker(const CoreWorkerOptions &options, const WorkerID &worker_
       resource_ids_(new ResourceMappingType()),
       grpc_service_(io_service_, *this),
       task_execution_service_work_(task_execution_service_) {
-  RAY_LOG(DEBUG) << "Constructing CoreWorker, worker_id: " << worker_id;
+  RAY_LOG(DEBUG) << "Constructing CoreWorker, worker_id: " << worker_id
+                 << ", node_ip_address: " << options_.node_ip_address;
```

Comment:
Is the `options_.node_ip_address` may be a hostname? If so, we should refactor the variable name.

Reply:
So many places are using this name. I'm not sure whether to rename. @raulchen What do you think?

Reply:
Dashboard frontend also distinguishes hostname and ip. We also can not rename simply.